pkg/kube/inject/webhook.go

```diff
@@ -659,6 +664,10 @@ func toAdmissionResponse(err error) *v1beta1.AdmissionResponse {
 func (wh *Webhook) inject(ar *v1beta1.AdmissionReview, path string) *v1beta1.AdmissionResponse {
 	req := ar.Request
 	var pod corev1.Pod
+
+	// TODO: URGENT, invalid request testing for injector !!!!
+	// - catch throwables, don't crash the server !!!
```

Comment:
Its not that urgent as far as I know, crashes are handled gracefully. I don't think we *should* panic but its not a huge issue if we do?

Reply:
Right, we have a handler. Will update the comment - I just had the logs full of stack traces, didn't realize it was not restarting just filling the logs.

It is still important to check the params and just return a 400 - if the inject is exposed via Gateway, in particular since we don't have a good way to authenticate it's from the apiserver.